modules/post/windows/escalate/ms18_8120_win32k_privsec.rb

```diff
@@ -0,0 +1,86 @@
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Post::File
+  include Msf::Post::Windows::Registry
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Win32k Elevation of Privilege Vulnerability',
+      'Description' => %q{
+        This module exploits elevation of privilege vulnerability exists in Windows when the Win32k
+        component fails to properly handle objects in memory. An attacker who successfully exploited
+        this vulnerability could run arbitrary code in kernel mode. An attacker could then install
+        programs; view, change, or delete data; or create new accounts with full user rights.},
+      'References'  =>
+        [
+          ['CVE', '2018-8120'],
+          ['URL', 'https://portal.msrc.microsoft.com/en-US/security-guidance/advisory/CVE-2018-8120'],
+          ['URL', 'https://github.com/unamer/CVE-2018-8120']
```

Comment:
Looks like these references are also relevant:

* https://github.com/bigric3/cve-2018-8120
* http://bigric3.blogspot.com/2018/05/cve-2018-8120-analysis-and-exploit.html